sirepo/template/radia.py

```diff
@@ -1236,6 +1398,8 @@ def _update_geom_from_undulator(geom, und, beam_axis, height_axis):
     else:
         half_pole = _update_cuboid(
             half_pole,
+            None,
```

Comment:
What about making all of the `_update_*` functions take just kwargs and then you wouldn't have to do this? Also, there are many `def _update_...` functions and I don't think you dispatch to all of them. The ones that you do dispatch to should have a different name so it is clearer they are called through dispatch.